tdrs-frontend/src/__mocks__/axios.js

```diff
@@ -9,5 +9,10 @@ mockAxios.get = jest.fn(() =>
     data: {},
   })
 )
+mockAxios.post = jest.fn(() =>
```

Comment:
We need to declare this method in `__mocks__` to allow jest to mock POST requests over axios.